ext/sockets/tests/socket_set_option_in6_pktinfo.phpt

```diff
@@ -21,7 +21,11 @@ if (!defined('IPV6_PKTINFO')) {
 <?php
 
 $s = socket_create(AF_INET6, SOCK_DGRAM, SOL_UDP) or die("err");
-var_dump(socket_set_option($s, IPPROTO_IPV6, IPV6_PKTINFO, []));
+try {
+    var_dump(socket_set_option($s, IPPROTO_IPV6, IPV6_PKTINFO, []));
+} catch (\ValueError $e) {
+    echo $e->getMessage() . \PHP_EOL;
+}
```

Comment:
Added try but no change in output?

Reply:
I'll drop it, as I'm using WSL, I get test failures due to the protocols not being available so I've added it to catch the failure on CI, but seems like there is none.

Reply:
Ah no I remember why, this was when I still had the changes in conversion.c which I decided to drop